proxy/http2/Http2Stream.cc

```diff
@@ -509,7 +509,7 @@ Http2Stream::update_write_request(IOBufferReader *buf_reader, int64_t write_len,
         parent->connection_state.send_headers_frame(this);
 
         // See if the response is chunked.  Set up the dechunking logic if it is
-        is_done = this->response_initialize_data_handling();
+        this->response_initialize_data_handling();
```

Comment:
The boolean is not related to if `response_initialize_data_handling` was successful or not, which is bad coding in my opinion.  It is information about if the response was chunked and if it is finished.

The return value for `response_initialize_data_handling` should be changed.